mlcube/metrics_mlcube/example_custom_entrypoint/getting_started_3d_rad_seg.py

```diff
@@ -40,29 +41,29 @@ def create_csv(predictions, labels):
     input_data_df.to_csv("./data.csv", index=False)
 
 
-def run_gandlf(output_path, parameters_file):
+def run_gandlf(output_file, config):
     """
     A function that calls GaNDLF's generate metrics command with the previously created csv.
 
     Args:
-        output_path (str): The path to the output file/folder
-        parameters_file (str): The path to the parameters file
+        output_file (str): The path to the output file/folder
+        config (str): The path to the parameters file
     """
     exit_status = os.system(
-        f"python3.9 gandlf_generateMetrics -c {parameters_file} -i ./data.csv -o {output_path}"
+        f"gandlf generate-metrics -c {config} -i ./data.csv -o {output_file}"
     )
     exit_code = os.WEXITSTATUS(exit_status)
-    sys.exit(exit_code)
+    logging.error(exit_code)
```

Comment:
minor thing - are we sure it should be always logged as error? From what I understand this operation can be successfull too, so logging perhaps could depend on exit code value (i.e. info if 0 and error if 1)